main.cpp

```diff
@@ -31,14 +31,27 @@
 #define MBED_CONF_APP_ESP8266_RX MBED_CONF_APP_WIFI_RX
 #include "easy-connect/easy-connect.h"
 
+#ifdef TARGET_STM
```

Comment:
Just a generic note - I think we really need to start thinking how to handle these? I we pile them up in the main.cpp, it will become huge. Do we need to re-factor the code a bit and have some kind of generic LEDs in the main.cpp and then have some board.h where we define the buttons and leds per board?